org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/compiler/JvmModelGenerator.xtend

```diff
@@ -191,6 +191,15 @@ class JvmModelGenerator implements IGenerator {
 		appendable.decreaseIndentation.newLine.append('}')
 	}
 	
+	protected def (ITreeAppendable)=>ITreeAppendable memberSeparator() {
+		[
+			ITreeAppendable it |
+			// avoid generating empty lines with just two spaces
+			// https://github.com/eclipse/xtext-extras/issues/772
```

Comment:
I'm not sure that the comment here is necessary since `blankLine` pretty much says what it's supposed to do.